### ISSUE TITLE
Redirect to page without trailing slash loads entire page

### ISSUE THREAD

**Issue by danrot** [sulu-io/sulu#1027]
See discussion in https://github.com/sulu-io/sulu/pull/945#issuecomment-94721967

**Comment by mordamax** [comment]
I also noticed that if you create a page /about
then rename it to /about2
then rename it to /about3

and try to go to /about
you'll see chain of redirects /about 301 ->  /about2 301 -> /about3
which is bad for SEO
all change history should end up with 1 hop of redirect. In this case:

GET /about 301 -> /about3
GET /about2 301 -> /about3

https://take.ms/sJ59D
![image](https://user-images.githubusercontent.com/1177472/74611883-bebf0780-5108-11ea-9c23-512ef920f4ba.png)

**Comment by alexander-schranz** [comment]
@mordamax Thank you for report. Can you create a seperate issue for this. As this seems not be to this issue.

**Comment by danrot** [comment]
@mordamax sulu.rocks is an outdated version, can you try with the latest one as well? Just tested this on Sulu 2.0, and the issue as you describe it does not seem to occur there...

**Comment by mordamax** [comment]
@danrot  thanks, re-checked, it works fine

**Comment by nnatter** [comment]
Can we close this then? 😇

**Comment by danrot** [comment]
@nnatter No, the initial issue was about something else... The thing is that we load the entire page before doing the redirect, which is unnecessarily slow...
